### ISSUE TITLE
Chatbot request handler

### ISSUE THREAD

**Issue by drjakob** [roschaefer/story.board#247]
As a reporter,
I want the chatbot request handler (#241) to answer every chatfuel request. So if there is no trigger active there should output the standard-reply "Sorry, da stehe ich gerade auf dem Schlauch und habe keine Daten."
because every user question should be answered.

**Comment by drjakob** [comment]
Think API is what we need.

**Comment by drjakob** [comment]
Is this already closed?

**Comment by roschaefer** [comment]
Good point. We need a cucumber test for that situation. I guess we just send nothing. @leifg ?
